src/providers/oracle/qgsoraclesourceselect.h

```diff
@@ -134,7 +134,7 @@ class QgsOracleSourceSelect : public QDialog, private Ui::QgsDbSourceSelectBase
     //!Sets a new regular expression to the model
     void setSearchExpression( const QString& regexp );
 
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
+    void on_buttonBox_helpRequested() { QgsHelp::openHelp( QStringLiteral( "working_with_vector/supported_data.html#id2" ) ); }
```

Comment:
I think the main question to the doc is how to maintain a stable URI-schema (which is basically to be treated as an API).

* Either the current addresses will remain stable.
* Or there needs to be a mapping of stable addresses to resources (redirect mapping on the server)

What would be also nice is a travis check for dead links. Maybe have somewhere a list online (generated from QGIS master builds) with links which are in use. And join a second list generated from doc master builds. Where there is no matching doc, show it in red.
It's quite hard to get this right because doc and code are moving targets and there might be pull requests and stable releases involved.